analyzer.py

```diff
@@ -8,11 +8,15 @@
 from propertysuggester.utils.CompressedFileType import CompressedFileType
 
 if __name__ == "__main__":
-    logging.basicConfig(level=20) # Print logging.info
+    logging.basicConfig(level=20)  # Print logging.info
 
-    parser = argparse.ArgumentParser(description="this program generates a correlation-table from the csv-dump")
-    parser.add_argument("input", help="The CSV input file (wikidata triple)", type=CompressedFileType('rb'))
-    parser.add_argument("output", help="The CSV output file (default=sys.stdout)", default=sys.stdout, nargs='?',
+    parser = argparse.ArgumentParser(
+        description="this program generates a correlation-table from the csv-dump")
```

Comment:
This line is indented 4 spaces, similar lines below are indented much more. Can this be made consistent?

Reply:
In python, when the first arguments has to go to the next line (per PEP8) it should gets indented four spaces but when it's the second or more argument, it should go until the function call starts the "(" character. This is the way python works and doing otherwise makes PEP8 unhappy.

Reply:
All I say is that I do not like the code looking like this:
```Python
object.methodCall1("…",
                   param1="…",
                   param2="…")
object.methodCall2(
    param1="…",
    param2="…")
```
Please stick to either the style with each argument on a separate line, including the first:
```Python
object.methodCall1(
    "…",
    param1="…",
    param2="…")
```
… or stick to the other style:
```Python
object.methodCall2(param1="…",
                   param2="…")
```
Note that this is not meant to block this patch from being merged, but my question about the `from … import` lines is.

Reply:
Regarding import, they all are unused and that's and if bring them back flake8 starts to error out as unused variable.

Regarding the consistency, as a python developer I see the current way consistent,  put as many as possible in each line.

Reply:
Code like
```Python
    parser.add_argument("output",
                        help="The CSV output file (default=sys.stdout)",
                        default=sys.stdout, nargs='?',
                        type=CompressedFileType('wb'))
```
does not look like the goal was to "put as many as possible in one line". If the line length really is 120, this should more look like
```Python
    parser.add_argument("output", help="The CSV output file (default=sys.stdout)", default=sys.stdout, nargs='?',
                        type=CompressedFileType('wb'))
```
I hope this explains better where my confusion comes from.

Reply:
Oh, because it should be 80 (and not 120) but long lines are too much to be done in one go so I fixed very big ones first and I'm reducing it to 100 (and then 80)